dictionaries/telemetry/damageCauserName.json

```diff
@@ -6,6 +6,9 @@
   "BattleRoyaleModeController_Desert_C": "Bluezone",
   "BattleRoyaleModeController_Savage_C": "Bluezone",
   "Boat_PG117_C": "PG-117",
+  "BP_M_Rony_A_01_C": "Pickup Truck (open top)",
+  "BP_M_Rony_A_02_C": "Pickup Truck (open top)",
+  "BP_M_Rony_A_03_C": "Pickup Truck (open top)",
```

Comment:
If they are all open top, you can probably just leave that off.

Reply:
Also, is pickup truck the accepted reference to the rony, or do people call it the rony?

Reply:
It's a pickup truck that's also called the Rony.. Rony might be better here though. I'll update it.